### ISSUE TITLE
Biomes not registering always due to BA+ loading before Biome mods sometimes

### ISSUE THREAD

**Issue by Sunconure11** [itsmeow/betteranimalsplus#47]
More specifically, support to spawn your mobs in their biomes. Biomes O' Plenty appears to have been done by default, at least, via the config.

**Comment by Sunconure11** [comment]
I'm reopening this, for I've noticed in the pack used on the bewitchment server, the mobs do not show up as valid for the biomes in PVJ, in the config that is.

**Comment by itsmeow** [comment]
All config values are valid, but if a biome id does not exist it will give an error. Is it giving errors for PVJ?

**Comment by Sunconure11** [comment]
I don't see any errors. But as we conversed in staff on the BW channel, we probably found the cause, and need to load this mod after all others to get them supported.

```
Load after the following mods:

Project Vibrant Journeys
Traverse
Novam Terram
Plants 2
Twilight Forest
Bewitchment (I DO have plans for a few biome ideas at some point)
Oh The Biomes You'll Go
ChinjufuMod
Sugi Forest
Zoesteria Biomes
Vampirism
Defiled Lands
Abyssalcraft
```

**Comment by Sunconure11** [comment]
I don't see anything out of the ordinary in my log, either.

Also, just for safe measure, make sure JEID and NEID don't somehow screw with stuff, since both allow for more biome IDs.

**Comment by itsmeow** [comment]
Note for self: Even better fix would be to register spawns on world load (FMLServerStartingEvent) if they are not already loaded

**Comment by Shybella** [comment]
This also appears to be an issue with OTG + Biomes Bundle.

Goats and Pheasants only seem to be spawning.

**Comment by itsmeow** [comment]
Also causes this if manually adding to configs:
![image](https://user-images.githubusercontent.com/10366817/55833841-7bf2d300-5ae6-11e9-8399-0f57dbb2a294.png)
I will be releasing a 5.0.2 patch to fix this as soon as I get it working- 6.0.0 will take a while as it is a very large update.

**Comment by itsmeow** [comment]
You will need to delete and allow your config to regenerate to resolve this in existing servers. Update releasing soon.
